### ISSUE TITLE
Spec output is non-deterministic

### ISSUE THREAD

**Issue by crmilsap** [apiflask/apiflask#373]
Running `flask spec --output openapi.yaml` repeatedly results in output files with diffs when working on a large project. This is problematic when consuming the output file to generate an HTTP client.

Environment:

- Python version: 3.9.13
- Flask version: 2.0.3
- APIFlask version: 1.1.3

**Comment by greyli** [comment]
Hi, what are the diffs? If you update the views/schemas, the spec should be different.

**Comment by crmilsap** [comment]
> Hi, what are the diffs? If you update the views/schemas, the spec should be different.

There are diffs when making no changes to the schemas. The orders of parameters change unnecessarily.

**Comment by greyli** [comment]
Thanks. This should be fixed with #387.

I will make a new release no later than 31 December. Before that, you can keep the schema in order by adding a base schema for all your schemas:

```
from apiflask import Schema

class OrderedSchema(Schema):
    class Meta:
        ordered = True


class PetSchema(OrderedSchema):
    pass
```
